lib/src/cli/deployapps/baas_client.dart

```diff
@@ -90,21 +97,16 @@ class BaasClient {
         result[app.name] = app;
       }
     }
-    String appName = "flexible";
-    if (!result.containsKey(appName)) {
-      result[appName] = await _createApp(appName);
-    }
-    appName = "autoConfirm";
-    if (!result.containsKey(appName)) {
-      result[appName] = await _createApp(appName, confirmationType: 'auto');
-    }
+    await _createAppIfNotExists(result, defaultAppName);
+    await _createAppIfNotExists(result, "autoConfirm", "auto");
+    await _createAppIfNotExists(result, "emailConfirm", "email");
+    return result;
+  }
 
-    appName = "emailConfirm";
+  Future<void> _createAppIfNotExists(Map<String, BaasApp> result, String appName, [String? confirmationType]) async {
```

Comment:
[nit] I know that the variable is called `result` in the calling method, but here it's probably better to use `existingApps` or something along these lines.

Reply:
Done